packages/runtime/src/util-methods.ts

```diff
@@ -15,3 +16,20 @@ export function mountUtilMethods() {
     }
   });
 }
+
+// parse component Type
+export function parseType(v: string) {
```

Comment:
This file was designed for 'util methods can be invoked in the expression', like `{{ alert('hi') }}`, sorry for the misleading naming.

So I think parseType is a util function for this package?

Reply:
Yes. Now I create a new folder for utils functions.